service/create/service.go

```diff
@@ -154,13 +188,13 @@ func (s *Service) Boot() {
 					}
 
 					// Run masters
-					if err := s.runMachines(cluster.Spec.Masters, cluster.Name, "master"); err != nil {
+					if err := s.runMachines(cluster.Spec, cluster.Name, prefixMaster); err != nil {
 						s.logger.Log("error", microerror.MaskAny(err))
 						return
 					}
 
 					// Run workers
-					if err := s.runMachines(cluster.Spec.Workers, cluster.Name, "worker"); err != nil {
+					if err := s.runMachines(cluster.Spec, cluster.Name, prefixWorker); err != nil {
```

Comment:
I am quite bad at separating work. When I see something I quite often change it immediately and mix different things in the same PR. So this prefix changes could be provided separately. Please try to separate your work based on the concerns of the changes you drive in the future.